### ISSUE TITLE
New Dynamic pages

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#869]
Some pages we could automate and save Dave a little time!

- [ ] [Dev Roundtable](https://galaxyproject.org/community/devroundtable/)
- [ ] [Galaxy Newsletter](https://galaxyproject.org/galaxy-updates/)

**Comment by tnabtaf** [comment]
Yes!  My intent was to automate the Roundtable at some point to help me learn how things work.  However, whatever I did would be a copy and paste hack.  I'm not sure if the newsletter page has a good reason to exist anymore.  It is largely redundant with the news page.
